lib/android-helpers.js

```diff
@@ -335,6 +335,14 @@ helpers.initUnicodeKeyboard = async function (adb) {
   return defaultIME;
 };
 
+helpers.setMockLocationApp = async function (adb, app) {
+  if (await adb.getApiLevel() < 23) {
```

Comment:
yes, there's nothing wrong with what you're doing. technically it's OK since JS does implicit conversion when comparing strings and numbers, but i never like to rely on that kind of implicit stuff. i'm just wondering if we should take the opportunity now to be more explicit, since we're looking at it.